### PR TITLE
Correct errors and make clarifying edits for remote administration

### PR DESCRIPTION
**Pull request by wallacekelly-da**
This section has never been clear to me. I've made what I believe are clarifying edits.

### CODE REVIEW

**Comment by marciw-da** [comment]
Can you find a willing reviewer on the Canton team and add them to this PR? Thanks!

**Comment by wallacekelly-da** [comment]
Shopping for a willing reviewer [here](https://daholdings.slack.com/archives/C973Z3TK2/p1710883068855399).

**Comment by garyverhaegen-da** [comment]
It would make me so happy if we could eschew the `bin/canton` script and just describe all of this work directly in terms of `java -jar canton-enterprise-x.y.z.jar` instead.

**Comment by wallacekelly-da** [comment]
> It would make me so happy if we could eschew the `bin/canton` script and just describe how all of this works directly in terms of `java -jar canton-enterprise-x.y.z.jar` instead.

That sounds like a bigger issue than what this PR is trying to accomplish.

File: docs/2.9.0/docs/canton/usermanual/console.rst
Comment (marciw-da) [inline docs/2.9.0/docs/canton/usermanual/console.rst:40]:
```suggestion
The Canton console works with both
local in-process nodes and remote nodes.
Once you've configured the network address, port, and token information,
you can use a single Canton console to administer all the nodes in your system.
```

File: docs/2.9.0/docs/canton/usermanual/console.rst
Comment (marciw-da) [inline docs/2.9.0/docs/canton/usermanual/console.rst:77]:
```suggestion
You might also want to consolidate the auto-generated configurations into a single remote configuration file.
```

Comment (marciw-da) [reply docs/2.9.0/docs/canton/usermanual/console.rst:77]: We follow the "may" guidance here: https://developers.google.com/style/word-list#may

File: docs/2.9.0/docs/canton/usermanual/console.rst
Comment (marciw-da) [reply docs/2.9.0/docs/canton/usermanual/console.rst:40]: I'm just trying to clarify/simplify a bit, but feel free to veto these edits

File: docs/2.9.0/docs/canton/usermanual/console.rst
Comment (marciw-da) [inline docs/2.9.0/docs/canton/usermanual/console.rst:67]:
```suggestion
Note that most Canton commands can be executed from a remote console.
However, a few commands can only be called
from the local console of the node itself.
```

**Review by marciw-da — APPROVED** [review]
Thanks! Approving with minor suggestions. (The suggestions are in the 2.9.0 file but apply to all the other versions -- but you can skip 3.0.0 because it will be re-created afresh from 2.9.0)

**Comment by wallacekelly-da** [comment]
Rebased on `main` and resolved conflict

**Comment by garyverhaegen-da** [comment]
We have [started the 3.1 documentation folder "for real"](https://github.com/digital-asset/docs.daml.com/pull/675). This pull requests makes changes to 2.9.0; please consider whether this should be forward-ported to 3.1 before merging it.

**Comment by wallacekelly-da** [comment]
Rebased on `main` and resolved conflicts to get this on the 3.1 docs.

File: docs/2.7.9/docs/canton/usermanual/console.rst
Comment (soren-da) [inline docs/2.7.9/docs/canton/usermanual/console.rst:77]: You can actually also pass in multiple config files into canton `-c first.conf,second.conf`

**Review by soren-da — APPROVED** [review]
LGTM

File: docs/2.7.9/docs/canton/usermanual/console.rst
Comment (wallacekelly-da) [reply docs/2.7.9/docs/canton/usermanual/console.rst]: Great suggestion! Thanks 19d4ee6

**Comment by wallacekelly-da** [comment]
This is ready to merge! 🎉  Waiting for CI/CD jobs.

**Pull request merged**
